moveit_experimental/moveit_jog_arm/src/jog_interface_base.cpp

```diff
@@ -94,7 +94,7 @@ bool JogInterfaceBase::readParameters(ros::NodeHandle& n)
   error += !rosparam_shortcuts::get("", n, parameter_ns + "/planning_frame", ros_parameters_.planning_frame);
   error += !rosparam_shortcuts::get("", n, parameter_ns + "/use_gazebo", ros_parameters_.use_gazebo);
   error += !rosparam_shortcuts::get("", n, parameter_ns + "/check_collisions", ros_parameters_.check_collisions);
-  error += !rosparam_shortcuts::get("", n, parameter_ns + "/warning_topic", ros_parameters_.warning_topic);
+  error += !rosparam_shortcuts::get("", n, parameter_ns + "/status_topic", ros_parameters_.status_topic);
```

Comment:
Changes to rosparam names using rosparam_shortcuts can break applications that use this. You should use the node handle interface to check for "/status_topic" and if it exists use the value. If it doesn't exist then print a warning about the parameter name change and check for "/warning_topic" and use that if it is found. If it is not found, you can print an error and increment `error`.

Reply:
done